src/components/common/Chart/Chart.tsx

```diff
@@ -98,71 +98,76 @@ function ChartContent(props: ChartContentProps): JSX.Element {
   const canvasRef = useRef<HTMLCanvasElement>(null);
   const [chart, setChart] = useState<ChartJS | null>(null);
   const theme = useTheme();
+  const initialized = useRef(false);
```

Comment:
Will this be reset to false when the component's props change?

Reply:
It is not reset, but the only prop that we want to re-trigger the useEffect (where this _"initialized"_ variable is used) is locale, and when changing locale, charts re-renders correctly (I don't know why)